.github/workflows/ci.yml

```diff
@@ -459,17 +556,135 @@ jobs:
       with:
         jobs: ${{ toJSON(needs) }}
 
+  github-release-pull-request:
+    name: >-
+      Open a GitHub Pull Request for
+      ${{ needs.pre-setup.outputs.git-tag }}
+    if: fromJSON(needs.pre-setup.outputs.release-requested)
+    needs:
+    - check
+    - pre-setup  # transitive, for accessing settings
+
+    runs-on: Ubuntu-latest
+
+    environment: git-repo
+
+    outputs:
+      pull_request_url: ${{ steps.pr.outputs.pull_request_url }}
+
+    permissions:
+      pull-requests: write
+
+    steps:
+    - name: Fetch the src
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # To reach the common commit w/ the repo default branch
+        ref: ${{ github.event.inputs.release-commitish }}
+        ssh-key: ${{ secrets.DEPLOYMENT_KEY }}
+    - name: Setup git user as [bot]
+      # Refs:
+      # * https://github.community/t/github-actions-bot-email-address/17204/6
+      # * https://github.com/actions/checkout/issues/13#issuecomment-724415212
+      uses: fregante/setup-git-user@v1.0.1
+    - name: Fetch the GHA artifact with the version patch
+      uses: actions/download-artifact@v2
+      with:
+        name: npm-package-bump-patch
+
+    - name: Apply the version patch
+      run: git am ${{ needs.pre-setup.outputs.version-patch-name }}
+    - name: Drop the version patch file
+      run: rm -fv ${{ needs.pre-setup.outputs.version-patch-name }}
```

Comment:
Forgot to apply a changelog patch here. Will be fixed by https://github.com/ansible/ansible-language-server/pull/202.